app.feeddeck.feeddeck.yml

```diff
@@ -0,0 +1,46 @@
+app-id: app.feeddeck.feeddeck
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: feeddeck
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=home
+
+modules:
+  - name: FeedDeck
+    buildsystem: simple
+    build-commands:
+      - cp -R bundle /app/feeddeck
+      - mkdir -p /app/bin && ln -s /app/feeddeck/feeddeck /app/bin/feeddeck
+      - chmod +x /app/bin/feeddeck
+      - install -Dm644 app.feeddeck.feeddeck.svg /app/share/icons/hicolor/scalable/apps/app.feeddeck.feeddeck.svg
+      - install -Dm644 app.feeddeck.feeddeck.desktop -t /app/share/applications/
+      - install -Dm644 app.feeddeck.feeddeck.metainfo.xml -t /app/share/metainfo/
+    sources:
+      - type: file
+        url: https://raw.githubusercontent.com/feeddeck/feeddeck/main/app/linux/flatpak/app.feeddeck.feeddeck.desktop
+        sha256: c66ab88c9683b69914e12d4bc780a46473631b2539ca83503fae404145eae7cf
+      - type: file
+        url: https://raw.githubusercontent.com/feeddeck/feeddeck/main/app/linux/flatpak/app.feeddeck.feeddeck.metainfo.xml
+        sha256: 6c818c36163ebffddea746bea2080bf6fbd571b5eb9f85a0074757b1106d1614
```

Comment:
Wrong checksum here

Reply:
This should be fixed in https://github.com/flathub/flathub/pull/4568/commits/701b65f69601c63c2b95bce065f991a59b2a9959